examples/test-secrets-in-explicit-provider/csharp/Program.cs

```diff
@@ -27,7 +27,7 @@
    });
 
    // Test handling dynamic secrets that start as unknown.
-   var providerWithSecretUsername = new Pulumi.Docker.Provider("provider-with-secret-username", new Pulumi.Docker.ProviderArgs
+   var providerWithSecretUsername = new Pulumi.Docker.Provider("provider-with-sensitive-username", new Pulumi.Docker.ProviderArgs
```

Comment:
"secret-username" as part of the name trips up the asserts that search for that string verbatim in the state.